network_analysis/find-connected-features-in-utility-network/README.md

```diff
@@ -2,46 +2,47 @@
 
 ## How it works
 
-1. Create a `MapView` and subscribe to its `GeoViewTapped` event.
-2. Create and load a `Map` that contains `FeatureLayer`(s) that are part of a utility network.
+1. Create a `Map` and add it to a `MapView`.
+2. Using the URL to a utility network's feature service, create `FeatureLayer`s that contain the utility network's features, and add them to the operational layers of the map.
 3. Create and load a `UtilityNetwork` with the same feature service URL and map.
 4. Add a `GraphicsOverlay` with symbology that distinguishes starting points from barriers.
-5. Identify features on the map and add a `Graphic` that represents its purpose (starting point or barrier) at the location of each identified feature.
-6. Determine the type of the identified feature using `UtilityNetwork.Definition.GetNetworkSource` passing its table name.
-7. If a junction, display a terminal picker when more than one terminal is found and create a `UtilityElement` with the selected terminal or the single terminal if there is only one.
-8. If an edge, create a `UtilityElement` from the identified feature and compute its `FractionAlongLine` using `GeometryEngine.FractionAlong`.
-9. Run a `UtilityNetwork.TraceAsync` with the specified parameters.
-10. Group the `UtilityElementTraceResult.Elements` by `NetworkSource.Name`
-11. For every `FeatureLayer` in this map with elements, select features by converting `UtilityElement`(s) to `ArcGISFeature`(s) using `UtilityNetwork.GetFeaturesFromElementsAsync`
+5. Add a listener for clicks on the map view, and use `mapView.identifyLayersAsync()` to dentify clicked features.
+6. Add a `Graphic` that represents its purpose (starting point or barrier) at the location of each identified feature.
+7. Determine the type of the identified feature using `utilityNetwork.getDefinition().getNetworkSource()` passing its table name.
+8. If a junction, display a terminal picker when more than one `UtilityTerminal` is found and create a `UtilityElement` with the selected terminal or the single terminal if there is only one.
+9. If an edge, create a utility element from the identified feature and set its `FractionAlongEdge` using `GeometryEngine.fractionAlong()`.
+10. Create `UtilityTraceParameters` for the trace, specifying the trace type, `UtilityTraceType.CONNECTED`, adding the starting location, and adding barriers (if applicable).
+11. Run a `utilityNetwork.traceAsync()` with the specified parameters, and get the `UtilityTraceResult`.
+12. From the utility trace result, get the `UtilityElementTraceResult`, and group the utility elements within by their network source name, which you can retrieve using `utilityElement.getNetworkSource().getName()`.
```

Comment:
we're not using first person throughout the rest of the readme, so I think here we could say:

> which can be retrieved using `utilityElement.getNEtworkSource().getName()`